platform/admin/ui/src/main/webapp/js/views/configuration/ConfigurationEdit.view.js

```diff
@@ -247,9 +247,13 @@ define([
                             case 'ViewToModel':
                                 return bindValue.toString();
                             case 'ModelToView':
-
+                                var bindValueString = "";
                                 if(bindValue && bindValue !== ""){
```

Comment:
I believe the blank string is also falsey, so we can remove the second part of that conditional clause.